streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -698,7 +698,7 @@ void to(final Serde<K> keySerde,
      * @see #mapValues(ValueMapper)
      * @see #transform(TransformerSupplier, String...)
      */
-    <VR> KStream<K, VR> transformValues(final ValueTransformerSupplier<? super V, ? extends VR> valueTransformerSupplier,
+    <VR> KStream<K, VR> transformValues(final ValueTransformerSupplier<? super V, VR> valueTransformerSupplier,
```

Comment:
I don't think this one needs to change. AFAICT it is only `transform` that has the issue